docs/governance/templates/release_issue.md

```diff
@@ -58,8 +58,8 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] Attach all assets found in the cloud storage with {version} to the draft GitHub Release.
 - [ ] Copy any review changes from the release blog post into the draft GitHub release.
 - [ ] Publish the draft GitHub Release.
-- [ ] Run `make release-branch` to create a release branch.
-- [ ] Email the [mailing list][list] with the release details (copy-paste the release blog post).
+- [ ] Run `make release-branch` to create a release branch and run `gcloud config configurations activate <your development project>` to switch Agones development tooling off of the `agones-images` project.
+- [ ] Email the `mailing list` with the release details (copy-paste the release blog post). Refer [Internal Mailing list posting](go/agones-release-guide). 
```

Comment:
Minor nits on grammar:

```suggestion
- [ ] Email mailing lists with the release details (copy-paste the release blog post). Refer to the [Internal Mailing list posting guide](go/agones-release-guide) for details.
```

Other than that, LGTM.

Reply:
Done!